Library/Homebrew/formulary.rb

```diff
@@ -1201,7 +1201,6 @@ def self.find_formula_in_tap(name, tap)
       "#{name}.rb"
     end
 
-    Tap.formula_files_by_name(tap)
-       .fetch(name, tap.formula_dir/filename)
+    tap.formula_files_by_name.fetch(name, tap.formula_dir/filename)
```

Comment:
```suggestion
    tap.formula_files_by_name.fetch(name) { tap.formula_dir/filename }
```

To avoid building unnecessary `Pathname` objects when the token is in the hash.

Reply:
@apainintheneck same deal as last PR: would like to see something like `brew prof` show a measurable impact before doing these sorts of micro-optimisations.